### PR TITLE
SASS-6159: J-3 Create 'UK interest' page - Income from property, pens…

### PR DESCRIPTION
**Pull request by AishaBi**
…ions and investments - TYR II

### Description
Please include a brief summary of the change you've made and the issue that it fixes.

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the it tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?
- [ ]  Have you done the group review in the team?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [ ]  Have you run the it tests?
- [ ]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- [ ]  Have you checked for individual and agent content matching with the prototype? (where applicable)
- [ ]  Have you checked for welsh transaltion for individual and agent? (where applicable)
- [ ]  Have you checked for errors messages for individual and agent on both english and welsh? (where applicable)
- [ ]  Run wave (chrome extension) for accessibility (where applicable)

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?
- [ ]  Demo to Business Analyst

### CODE REVIEW

**Comment by rhysconnor** [comment]
Can we add `override def path: JsPath = JsPath \"propertyPensionsInvestments"\ toString` to Pages.UkInterestPage

File: conf/messages.en [hunk 1]
```diff
@@ -478,3 +463,40 @@ paymentsIntoPensions.checkYourAnswersLabel = paymentsIntoPensions
...
+ukInterest.fromUkBanks = Interest from UK banks and building societies
+ukInterest.fromUkTrustFunds = Interest from UK trust funds or bonds
+ukInterest.fromGiltEdged = Interest from gilt-edged or accrued income securities
+ukInterest.noInterest = No, I did not get any income from UK interest
+
+ukInterest.agent.fromUkBanks = Interest from UK banks and building societies
+ukInterest.agent.fromUkTrustFunds = Interest from UK trust funds or bonds
+ukInterest.agent.fromGiltEdged= Interest from gilt-edged or accrued income securities
+ukInterest.agent.noInterest = No, my client did not get any income from UK interest
+ukInterest.error.required = Select if you had income from UK interest, or No, you did not get any income from UK interest
```
Comment (rhysconnor) [inline conf/messages.en:501]: The errors for this has been updated please see comment section of the parent ticket

File: conf/messages.en [hunk 2]
```diff
@@ -478,3 +463,40 @@ paymentsIntoPensions.checkYourAnswersLabel = paymentsIntoPensions
 paymentsIntoPensions.error.required = Select paymentsIntoPensions
 paymentsIntoPensions.agent.error.required = Select paymentsIntoPensions
 paymentsIntoPensions.change.hidden = PaymentsIntoPensions
+
+###################################  UK interest ###########################################################################################
```
Comment (rhysconnor) [inline conf/messages.en:467]: I know the welsh content isnt ready yet but could we add the english messages into the welsh content just as placeholders?

File: app/models/propertypensionsinvestments/UkInterest.scala [hunk 3]
```diff
@@ -26,29 +26,39 @@ sealed trait UkInterest
...
 object UkInterest extends Enumerable.Implicits {
 
-  case object Option1 extends WithName("option1") with UkInterest
-  case object Option2 extends WithName("option2") with UkInterest
-  case object ExclusiveOption extends WithName("exclusive") with UkInterest
+  case object FromUkBanks extends WithName("fromUkBanks") with UkInterest
+  case object FromUkTrustFunds extends WithName("fromUkTrustFunds") with UkInterest
+  case object FromGiltEdged extends WithName("fromGiltEdged") with UkInterest
+  case object Divider extends UkInterest
+  case object  NoInterest extends WithName("noInterest") with UkInterest
```
Comment (rhysconnor) [inline app/models/propertypensionsinvestments/UkInterest.scala:33]: can we align NoInterest

**Review by rhysconnor — CHANGES_REQUESTED** [review]

**Review by ansahdavid — APPROVED** [review]

**Review by kannanramakrishnan — APPROVED** [review]

**Pull request merged**